kubevirt_scripts/Containerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.fedoraproject.org/fedora:39
```

Comment:
probably better not use the latest fedora here, maybe Fedora 38 or even a more stable distribution?
I'm just being conservative here

Reply:
Changed. These scripts are not supported anyway.